### PR TITLE
Log out important directories for validator client

### LINKED ISSUE #5617: Request: Display beaconchain.db and validator key paths at startup
Users usually copy/paste the commands (or doubleclick the binarys) and don't know where their .db & validator keys are located at, it would be very helpful to display the path at the beginning of the process.

### PR DESCRIPTION
**Pull request by ShawkiS**
Resolves #5617

Description
Write why you are making the changes in this pull request

I want to print the .db and validator keys location at the beginning of the process.

Write a summary of the changes you are making:
-Logging the DB path in validator client in `NewValidatorClient` in `validator/node/node.go`
-Logging the Keystore path in validator client in `NewKeystore` function in `validator/keymanager/direct_keystore.go`

### CODE REVIEW

File: slasher/db/kv/kv.go [hunk 1]
```diff
@@ -76,6 +77,7 @@ func NewKVStore(dirPath string, cfg *Config) (*Store, error) {
 	if err := os.MkdirAll(dirPath, 0700); err != nil {
 		return nil, err
 	}
+	logrus.Infof(".db & validator keys dir is: %v", dirPath)
```
Comment (0xKiwi) [inline slasher/db/kv/kv.go:80]: This isn't needed, the slasher already logs out the datadir.

File: validator/db/db.go [hunk 2]
```diff
@@ -68,6 +68,7 @@ func NewKVStore(dirPath string, pubKeys [][48]byte) (*Store, error) {
 	if err := os.MkdirAll(dirPath, 0700); err != nil {
 		return nil, err
 	}
+	log.Infof(".db & validator keys dir is: %v", dirPath)
```
Comment (0xKiwi) [inline validator/db/db.go:71]: This is not correct, the validator keys are not in this directory. You will need another log to declare where they are. Look at how the `Checking DB` log is put out and mirror that for this one.

File: slasher/db/kv/kv.go (hunk 1 above)
Comment (ShawkiS) [reply slasher/db/kv/kv.go:80]: So I keep the line for the .db?

File: validator/db/db.go [hunk 3]
```diff
@@ -68,6 +69,8 @@ func NewKVStore(dirPath string, pubKeys [][48]byte) (*Store, error) {
 	if err := os.MkdirAll(dirPath, 0700); err != nil {
 		return nil, err
 	}
+	log.WithField("database-path", dirPath).Info("Checking DB")
+	log.WithField("validator-keys-path", cmd.DataDirFlag.Name).Info("Checking Validator Keys")
```
Comment (prestonvanloon) [inline validator/db/db.go:73]: Wouldn't this print `validator-keys-path=datadir`?

Comment (prestonvanloon) [reply validator/db/db.go:73]: By this i mean, you are printing the flag name, not the value of the flag.

Comment (ShawkiS) [reply validator/db/db.go:73]: Oh sorry, What is the property I have to use to get the datadir?

Comment (rauljordan) [reply validator/db/db.go]: Hi @ShawkiS typically we can use something like `cliCtx.GlobalString(cmd.DataDirFlag.Name)`

File: validator/node/node.go [hunk 4]
```diff
@@ -101,6 +101,7 @@ func NewValidatorClient(ctx *cli.Context) (*ValidatorClient, error) {
 		if err := clearDB(dataDir, pubkeys, forceClearFlag); err != nil {
 			return nil, err
 		}
+		log.WithField("database-path and validator-keys-path", dataDir).Info("Checking DB and Validator Keys")
```
Comment (rauljordan) [inline validator/node/node.go:104]:
```suggestion
		log.WithField("path", dataDir).Info("Checking DB and Validator Keys")
```

Comment (ShawkiS) [reply validator/node/node.go]: Ok, thank you. I changed it.

File: validator/node/node.go [hunk 5]
```diff
@@ -101,6 +101,7 @@ func NewValidatorClient(ctx *cli.Context) (*ValidatorClient, error) {
 		if err := clearDB(dataDir, pubkeys, forceClearFlag); err != nil {
 			return nil, err
 		}
+		log.WithField("path", dataDir).Info("Checking DB path")
```
Comment (0xKiwi) [inline validator/node/node.go:104]: Can you mirror this to the beacon node log? So `Checking DB` and the field name `databasePath`.

File: validator/keymanager/direct_keystore.go [hunk 6]
```diff
@@ -45,6 +45,7 @@ func NewKeystore(input string) (KeyManager, string, error) {
 	if opts.Path == "" {
 		opts.Path = accounts.DefaultValidatorDir()
 	}
+	log.WithField("path", opts.Path).Info("Checking validator keystore path")
```
Comment (0xKiwi) [inline validator/keymanager/direct_keystore.go:48]: Same here, can you change the text to "Checking validator keys" and the field to "keystorePath"?

File: validator/node/node.go (hunk 5 above)
Comment (ShawkiS) [reply validator/node/node.go]: Okay, done

File: validator/keymanager/direct_keystore.go (hunk 6 above)
Comment (ShawkiS) [reply validator/keymanager/direct_keystore.go]: Okay, done

**Pull request merged**